common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -104,6 +104,7 @@ def _student_or_public_view(self, context, view):
             'due': self.due,
             'completed': completed,
             'past_due': past_due,
+            'has_graded_and_scored_children': completed is not None,
```

Comment:
You could also have the template check `completed is not None` but this is good

Reply:
I think I prefer the clarity this offers since we are going to want to migrate this to the MFE (probably, maybe, at some point) and I don't want us stuck and confused for a day on what the heck `completed is not None` means.

Reply:
OK. Well then an ignorable nit might be to rename it `has_assignments` because that's the vaguer word we use elsewhere to hide the exact definition (in case it changes).